XfelDispTest2.py

```diff
@@ -93,9 +93,9 @@ def setGOVal(self,ac, ac2):
         EndTimeForArchiver = val2.toPyDateTime().strftime('%m/%d/%Y %H:%M:%S')
         #print(StTimeForArchiver)
         if EndTimeForArchiver > StTimeForArchiver:   # Checks that the End time is after the Start time
-            #arPullGlob.getValuesOverTimeRange(StTimeForArchiver, EndTimeForArchiver)
+            arPullGlob.getValuesOverTimeRange(StTimeForArchiver, EndTimeForArchiver)
             #  Use Dummy value generator for now
-            arPullGlob.getValuesOverTimeDummy(StTimeForArchiver, EndTimeForArchiver)
```

Comment:
Could you put these in a conditional? Like, have a global bool DEBUG and have something like
`if DEBUG:
    arPullGlob.getValuesOverTimeDummy(StTimeForArchiver, EndTimeForArchiver)
else:
    arPullGlob.getValuesOverTimeRange(StTimeForArchiver, EndTimeForArchiver)
`

Reply:
agreed. Next edit.